### DISCUSSION TITLE
Column Equal to another Column

### DISCUSSION THREAD

**Discussion by andresmonc** [go-pg/pg#1903] [Q&A] [answered]
I'm trying to write a where clause where one column is equal to the value in another column however I'm getting an error 
ERROR #22P02 invalid input syntax for type integer: \"original_firecall_id\""

because it's putting the column value as a string and the id field is an int. Is there anyway to do this query with go-pg?
so the generated query is : 
```
 (id != ('original_firecall_id'))
```
but i'm looking for

```
(id != (original_firecall_id))
```
![image](https://user-images.githubusercontent.com/41450290/128104134-a8b38dac-1c68-4256-bfd8-dd54f9536696.png)

**Comment by vmihailenco** [comment, marked as answer]
Use `pg.Safe("original_firecall_id")`
